test/metabase/api/action_test.clj

```diff
@@ -38,49 +38,46 @@
 (deftest list-actions-test
   (mt/with-actions-enabled
     (mt/with-non-admin-groups-no-root-collection-perms
-      (mt/with-temp Card [{card-id :id} {:dataset true}]
-        (mt/with-model-cleanup [Action]
-          (let [posted-actions [{:name "Get example"
-                                 :type "http"
```

Comment:
There was an HTTP action being tested which is now missing. We definitely want to test keep testing every type because they have separate code paths